digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/util/DataFixture.scala

```diff
@@ -160,9 +168,12 @@ object DataFixture {
         recalculate()
       case Some ("update_missing") =>
         updateMissingRoadAddresses()
-      case Some("fuse_multi_segment_road_addresses") => {
+      case Some("fuse_multi_segment_road_addresses") =>
         combineMultipleSegmentsOnLinks()
-      }
+      case Some("update_road_addresses_geometry_staging") =>
```

Comment:
Why separate jobs and not just using the properties to figure out the environment?